### ISSUE TITLE
[BUG] Nuxt 3 Warning: Could not access `pt` in PrimeVue

### ISSUE THREAD

**Issue by Afrokk** [primefaces/primevue-tailwind#283]
Hello,

I'm currently using PrimeVue in a Nuxt 3 project and I've encountered a warning in the console related to the `pt` property.

**Description:**
When I run my Nuxt 3 application with PrimeVue, I get the following warning in the console:

```
Could not access `pt`. The only available runtime config keys on the client side are `public` and `app`.
```

This warning appears every time I load a page that uses a PrimeVue component. Everything else works fine.

Here's a snippet of my code:
```vue
<template>
    <div>
        <h1 class="text-4xl font-bold tracking-wide">Pipelines Visualization</h1>
        <p class="pt-2">Nuxt 3 + PrimeVue + Tailwind</p>
        <Button class="p-0 mt-4" @click="addEvent">Add Event</Button>
    </div>
</template>
```

> Using *any* `PrimeVue` component triggers this warning. 

Here's my `nuxt.config.ts`:
``` javascript
import path from 'path';

export default defineNuxtConfig({
  devtools: { enabled: true },
  css: ['~/assets/css/main.css','~/node_modules/primeicons/primeicons.css'],
  modules: ['nuxt-primevue', '@nuxtjs/tailwindcss'],
  primevue: {
    options: {
      unstyled: true,
    },
    importPT: { from: path.resolve(__dirname, './presets/my-preset/') }
  },
  nitro: {
    serverAssets: [
        {
            baseName: 'presets',
            dir: '../presets'
        }
    ]
}
})
```

**Expected behavior:**
I expect not to see any warnings in the console related to PrimeVue when running my Nuxt 3 application.

**Environment:**
- Nuxt version: 3.12.1
- PrimeVue version: 3.52.0 (Happens with v4 Beta 4 as well)
- Browser: Google Chrome

Thank you for your help with this issue.

**Comment by jinyongp** [comment]
This issue pertains to [PrimeVue](https://github.com/primefaces/primevue).

The issue has been resolved in [PrimeVue v4.0.0-rc.2](https://github.com/primefaces/primevue/releases/tag/4.0.0-rc.2), but it remains unresolved in version 3.

[Nuxt Discussion #27645](https://github.com/nuxt/nuxt/discussions/27645).

**Comment by jinyongp** [comment]
It'll be fixed in [v3.53](https://github.com/primefaces/primevue/milestone/182?closed=1).

I think it's okay to refer to [this](https://github.com/primefaces/primevue/commit/53e6246625afcca185037f49a8487d18474a363a) and solve it temporarily.

**Comment by Afrokk** [comment]
> It'll be fixed in [v3.53](https://github.com/primefaces/primevue/milestone/182?closed=1).
> 
> I think it's okay to refer to [this](https://github.com/primefaces/primevue/commit/53e6246625afcca185037f49a8487d18474a363a) and solve it temporarily.

Thank you!
